lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -110,6 +110,14 @@ class AutomateMethodException < StandardError
   require 'drb'
   require 'yaml'
 
+  module AutomateYamlLoadAliases
```

Comment:
Also super minor, but the only other thing I might do different is to just use an anonymous module here.  That way we can't possible "clobber" a user script that happens to have this module name (which is probably infinitely rare anyway)

Reply:
Something like

```ruby
YAML.singleton_class.prepend(
  Module.new do
    def safe_load(yaml, aliases: false, **kwargs)
      super(yaml, aliases: true, **kwargs)
    end
  end
)

Reply:
pushed, let me know if that's the direction you were thinking...

Reply:
pushed again, rubocop liked both ways, so I went with yours